auth-server/test/resolvers/mutation/addressLinkConfirm.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('addressLinkConfirm mutation', () => {
 
 	it('should not be able to confirm address link with fake signature', async () => {
 		const network = NetworkEnum.KUSAMA;
-		const address = 'HNZata7iMYWmk5RvZRTiAsSDhV8366zq2YGb3tLH5Upf74F'; // Alice
+		const address = 'FoQJpPyadYccjavVdTWxpxU7rUEaYhfLCPwXgkfD6Zat9QP'; // Bob
```

Comment:
We should delete the user and the address link in the DB after this test. I guess this is what conflicts with the other test.

Reply:
done